tsconfig.json

```diff
@@ -1,13 +1,17 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es6",
```

Comment:
Note that you can't safely target es6 unless you're on node 6 or greater.

I'd suggest holding off on this for this release. The move to node 6 minimum should probably at least be a minor version bump.